uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -4370,24 +4370,40 @@ public function testGetAttachmentsShouldThrowAnExceptionIfLeaveRequestIDIsMissin
   }
 
   public function testGetAttachments() {
```

Comment:
I wonder if we still need this test if with all new ones you've added. What does it have that the others don't?

Reply:
tbh, I wanted to remove it but the reason I left i was because the test checks for all the expected fields that the API is expected to return which the others don't. I could remove it though. What do you think?

Reply:
Maybe then it would make more sense to rename this method to better clarify that it is more about the fields? Or create some custom assert method that checks all the fields, reuse it in the other tests and delete this one.